XamlControlsGallery/ControlPages/NavigationViewPage.xaml

```diff
@@ -224,6 +224,56 @@
             </Grid>
         </local:ControlExample>
 
+        <local:ControlExample HeaderText="NavigationView with Footer Menu Items"
+                              VerticalAlignment="Top" 
+                              HorizontalContentAlignment="Stretch"
+                              XamlSource="NavigationView\NavigationViewSample9_xaml.txt"
+                              WebViewHeight="950">
```

Comment:
Not sure why WebViewHeight is set here and in other places. Does this behave correctly even if this is removed? I see it's on other samples on this page too, but I suspect it doesn't actually do anything.

Reply:
Not sure either. I removed it and all is fine.